charts/kafka-ui/values.yaml

```diff
@@ -84,17 +84,19 @@ probes:
   useHttpsScheme: false
 
 podSecurityContext:
-  {}
-  # fsGroup: 2000
+  fsGroup: 2000
 
 securityContext:
-  {}
-  # capabilities:
-  #   drop:
-  #   - ALL
-  # readOnlyRootFilesystem: true
-  # runAsNonRoot: true
-  # runAsUser: 1000
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+    - ALL
+  readOnlyRootFilesystem: true
```

Comment:
Can't merge this until https://github.com/kafbat/kafka-ui/issues/78 is completed. This might break the things for users with snappy. However, we can comment this out and merge the rest. What do you think @jcpunk ?

Reply:
I tried to rebase to sort of the conflicts and I think I've got this back mergeable.  Leaving the ROfs out for now makes sense.  I've set it to explicit "False" as a hint going forward.